### DISCUSSION TITLE
part-ii/2018-72/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#86] [Question comments]
Part II 2018 General Relativity: Paper 2, Section II, E 

https://questions.tripos.org/part-ii/2018-72/

**Comment by AndoBando** [comment]
Draw a picture, and then think about the dynamics of a particle in a 1d potential. The final case (iii) actually has two cases depending on the value of $\Lambda \mu^2$.
